content/en/blog/sign-verify-plugins-rules/index.md

```diff
@@ -0,0 +1,178 @@
+        "dev.sigstore.cosign/certificate": "-----BEGIN CERTIFICATE-----\nREDACTED\n-----END CERTIFICATE-----\n",
+        "dev.sigstore.cosign/chain": "-----BEGIN CERTIFICATE-----\nREDACTED=\n-----END CERTIFICATE-----\n-----BEGIN CERTIFICATE-----\nREDACTED\n-----END CERTIFICATE-----"
+      }
+    }
+  ]
+}
+```
+You can see that it contains a layer of type `application/vnd.dev.cosign.simplesigning.v1+json`. Also, its annotation metadata helps cosign to authenticate the signature against the Sigstore's [Rekor transparency log](https://docs.sigstore.dev/logging/overview/).
+
+#### Index update
+
+Each artifact type has its own [registry metadata file](https://github.com/falcosecurity/plugins/blob/master/registry.yaml). The general [falcoctl index.yaml](https://github.com/falcosecurity/falcoctl/blob/gh-pages/index.yaml), which refers to all artifact types, [is updated](https://github.com/falcosecurity/test-infra/blob/bc4db5255ad6f85ee437368bc37ee752f980182d/config/jobs/build-plugins/build-plugins.yaml#L2) when a modification to the specific artifact type metadata occurs. In order to verify a keyless signature we need to know in advance some additional metadata about the type of signature and how to verify its authenticity. We are adding this data to the registry metadata so that it can be published to the index by the pipeline. We will see this content in the next section.
+
+
+### Transparent signature verification in `falcoctl`
+
+![](images/falcoctl_signature_falco_part.png)
+
+We have seen that every artifact has now signature data available. This is nice, but to complete this feature we need to make sure that the signature data is automatically checked when we download an artifact. In the same way, package signatures are verified when you run `apt-get install`, falcoctl needs to do the same to the OCI artifacts it downloads.
+
+When designing this feature we decided not to change the existing usage of `falcoctl`. You know how when you decide to install a new repository for `apt` you need to also import its public key? Thanks to keyless signing trusting the Sigstore [Fulcio](https://docs.sigstore.dev/certificate_authority/overview/) CA we can avoid this step and make verification transparent. In the section [above](#Index-update) we introduced the concept of index files. Every time you use falcoctl, there is an index file that tells details about artifacts distributed by the Falco organization. For instance the `cloudtrail` plugin has [an entry](https://github.com/falcosecurity/falcoctl/blob/gh-pages/index.yaml#L15) with several pieces of metadata, including the OCI artifact coordinates: `ghcr.io/falcosecurity/plugins/plugin/cloudtrail`. Of course, anyone can distribute and [install their own index file](https://falco.org/blog/falcoctl-install-manage-rules-plugins/#index) for their plugins and rules.
```

Comment:
"You know how" maybe say "You may have experienced that when . . ."

Again maybe they don't know, so using "you may" includes folks who have not seen it before.

Reply:
I got it @mikegcoleman, and I agree. Thank you

Reply:
I've also rephrased a bit the first part of the paragraph:

```suggestion
When designing this feature we decided not to change the existing usage of `falcoctl`. For example, you may already know that when you decide to install a new `apt` repository you need to also import its public key. When installing Falco artifacts with `falcoctl`, we can avoid this step and make verification transparent, thanks to keyless signing by trusting the Sigstore [Fulcio](https://docs.sigstore.dev/certificate_authority/overview/) CA. In the section [above](#Index-update) we introduced the concept of index files. Every time you use falcoctl, there is an index file that tells details about artifacts distributed by the Falco organization. For instance, the `cloudtrail` plugin has [an entry](https://github.com/falcosecurity/falcoctl/blob/gh-pages/index.yaml#L15) with several pieces of metadata, including the OCI artifact coordinates: `ghcr.io/falcosecurity/plugins/plugin/cloudtrail`. Of course, anyone can distribute and [install their own index file](https://falco.org/blog/falcoctl-install-manage-rules-plugins/#index) for their plugins and rules.
```